src/rtScriptV8/rtScriptNode.cpp

```diff
@@ -1346,6 +1362,28 @@ rtError rtScriptNode::createContext(const char *lang, rtScriptContextRef& ctx)
   return RT_OK;
 }
 
+#ifdef HAVE_INSPECTOR
+rtError rtScriptNode::enableDebugger(bool enable, rtString host_name, int port)
+{
+  if (true == enable)
+  {
+    if (false == mDebuggerRunning) {
```

Comment:
I guess you should also account whether inspector was started in rtNodeContext::createEnvironment() . Please merge the latest code from master into your branch.